test/sagemaker_tests/pytorch/training/integration/sagemaker/test_mnist.py

```diff
@@ -26,20 +26,27 @@
 @pytest.mark.skip_gpu
 def test_mnist_distributed_cpu(sagemaker_session, framework_version, ecr_image, instance_type, dist_cpu_backend):
     instance_type = instance_type or 'ml.c4.xlarge'
-    _test_mnist_distributed(sagemaker_session, framework_version, ecr_image, instance_type, dist_cpu_backend)
+    _test_mnist_distributed(ecr_image, sagemaker_session, framework_version, instance_type, dist_cpu_backend)
```

Comment:
Why is this not run in multiple regions?

Reply:
added support for CPU